account-integrations/safe/src/SafeWebAuthnPlugin.sol

```diff
@@ -75,6 +57,9 @@ contract SafeWebAuthnPlugin is BaseAccount {
         return _publicKey;
     }
 
+    /** @notice validateSignature is called from Safe so the original function will always fail */
```

Comment:
Adding a comment for myself, verify exactly where validateSignature is getting called from